src/lib/clientOptions.ts

```diff
@@ -1,3 +1,19 @@
+/*
+ *  Copyright 2021 Curity AB
```

Comment:
2022, here and in the other files added in this PR or recently.

Reply:
Done